bin/windows-toolset.ps1

```diff
@@ -163,11 +151,29 @@ if ($buildSource) {
 cd $toolsDir
 
 # Download openssl
-Write-Host "Downloading http://buildsources.delivery.puppetlabs.net/windows/openssl/${opensslPkg}.tar.lzma"
-(New-Object net.webclient).DownloadFile("http://buildsources.delivery.puppetlabs.net/windows/openssl/${opensslPkg}.tar.lzma", "$toolsDir\${opensslPkg}.tar.lzma")
+Write-Host "Downloading https://s3.amazonaws.com/kylo-pl-bucket/${opensslPkg}.tar.lzma"
```

Comment:
We need these available for AppVeyor so we can run tests. Although this script is run internally only, we don't technically need them to be in a publicly accessible location, but my thought process was that it would be helpful to remember to update the s3 copies if we used them in this script.

cc/ @MikaelSmith